FlashFeetAPI/src/main/java/org/example/api/JobRoleService.java

```diff
@@ -0,0 +1,30 @@
+package org.example.api;
+
+import org.example.cli.JobRole;
+import org.example.client.FailedToGetJobsException;
+import org.example.db.DatabaseConnector;
+import org.example.db.JobRoleDao;
+
+import java.sql.Connection;
+import java.sql.SQLException;
+import java.util.List;
+
+public class JobRoleService {
+
+    private JobRoleDao jobRoleDao = new JobRoleDao();
+
+    public DatabaseConnector databaseConnector = new DatabaseConnector();
+
+    public JobRoleService(){
```

Comment:
[Possible Future Improvement] - Don't really need this. As this provided for you by the Object Class.